source/falaise/snemo/datamodels/sim_digi_data.cc

```diff
@@ -5,73 +5,131 @@
 
 namespace snemo {
 
-namespace datamodel {
+  namespace datamodel {
 
-// Serial tag :
-DATATOOLS_SERIALIZATION_SERIAL_TAG_IMPLEMENTATION(sim_digi_data, "snemo::datamodel::sim_digi_data")
+    // Serial tag :
+    DATATOOLS_SERIALIZATION_SERIAL_TAG_IMPLEMENTATION(sim_digi_data, "snemo::datamodel::sim_digi_data")
 
-sim_digi_data::sim_digi_data() {
-  _digi_events_.clear();
-  return;
-}
-
-sim_digi_data::~sim_digi_data() { return; }
+    sim_digi_data::sim_digi_data()
+    {
+      _calo_digi_hits_.clear();
+      _tracker_digi_hits_.clear();
+      _trigger_data_.clear();
+      _auxiliaries_.clear();
+      return;
```

Comment:
The `clear()` calls are superfluous here I think, as the implicit default construction should create the necessary initialization.

The `return` is also superfluous.